applications/user-api/src/main/java/no/fdk/userapi/LocalUserService.java

```diff
@@ -0,0 +1,67 @@
+package no.fdk.userapi;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class LocalUserService {
+    private String URL_BASE_STRING = "https://publishers-api.ut1.fellesdatakatalog.brreg.no/domains/";
+    private String URL_END = "/organizations";
+
+    User getUser(String id) {
+        String domain = id.substring(id.indexOf("@") + 1);
+        List<String> organisations = getOrganisationsAssociatedWithDomain(domain);
+
+        return new LocalUserAdapter(id, organisations);
+    }
+
+    private List<String> getOrganisationsAssociatedWithDomain(String domain) {
+        try {
+            URLConnection connection = new URL(URL_BASE_STRING + domain + URL_END).openConnection();
+            connection.setRequestProperty("Accept", "application/json");
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(connection.getInputStream()))) {
+                String jsonBody = reader.lines().collect(Collectors.joining(System.lineSeparator()));
+                ObjectMapper mapper = new ObjectMapper();
+                return mapper.readValue(jsonBody, List.class);
+            }
+        } catch (Exception e) {
+            return Collections.emptyList();
+        }
+    }
+
+    private class LocalUserAdapter implements User {
```

Comment:
Hvorfor må vi returnere som User?

Jeg ser at du må det for Altinn-brukerene, siden dette er slik du kontakter Altinn. Men nå er vi bare interessert i authorities, kan vi ikke bare ha et authorities endepunkt?

Reply:
vi hadde en veldig lang diskussjon jo i slack om hvis vi vil holde samme api  til alle brukerne (emailbrukerne har tomme navn) elle ha to forskjellige api - en til navne og ant til tilgang. Vi kan holde oss fra merge, til vi har kommet enige om API.

Med den nytt løsningide, foretrekker jeg å separere tilgang-api fra navn-api.